app/models/form/question.rb

```diff
@@ -174,6 +174,8 @@ def has_inferred_display_value?(case_log)
 
   def checkbox_answer_label(case_log)
     answer = []
+    return "Yes" if id == "declaration" && value_is_yes?(case_log["declaration"])
```

Comment:
Can this not just work the same as other checkbox questions? (i.e. 1 is automatically translated to Yes?